webpack.config.cjs

```diff
@@ -150,6 +150,17 @@ const sharedConfig = {
         use: [MiniCssExtractPlugin.loader, 'css-loader'],
         sideEffects: true,
       },
+      {
+        test: /\.(png|jpe?g|gif|webp)$/i,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              outputPath: 'images',
```

Comment:
```suggestion
              outputPath: '../images',
```
Currently, images are saved in the following path. assets/js/images. Changing the value to this will save the images in assets/images

Reply:
I thought about doing that but I don't think it actually adds any value.

Reply:
NB: this change would cause URLs to look like `https://web-stories.local/wp-content/plugins/web-stories-wp/assets/js/..images/631db33b339209fd6ab0bfa022326552.png`, so `publicPath` would need to be updated to remedy that. But I 'd prefer to not override the default `__webpack_public_path__` value.

Reply:
> I thought about doing that but I don't think it actually adds any value.

Images are currently stored in `assets/images`. Similarly, css is stored in it's own directory. Why would we change the path of images now?

Reply:
Now it's just a directory that's completely dynamic, can be completely excluded and never looked at The structure doesn't really matter.